frontend/src/features/mapSideBar/SidebarContents/LandForm.tsx

```diff
@@ -153,17 +165,39 @@ const Form: React.FC<ILandForm> = ({
               handlePinChange={handlePinChange}
               isAdmin={isAdmin}
               nameSpace="data"
+              isViewOrUpdate={isViewOrUpdate}
+              disabled={disabled}
             />
           </div>
         );
       case LandSteps.USAGE:
         return (
           <div className="parcel-usage">
-            <LandUsageForm classifications={classifications} nameSpace="data" {...formikProps} />
+            <LandUsageForm
+              classifications={classifications}
+              nameSpace="data"
+              {...formikProps}
+              disabled={disabled}
+            />
           </div>
         );
       case LandSteps.VALUATION:
-        return <LandValuationForm title="Bare Land Valuation" nameSpace="data" />;
+        return (
+          <LandValuationForm title="Bare Land Valuation" nameSpace="data" disabled={disabled} />
+        );
+      case LandSteps.ASSOCIATED_OR_REVIEW:
```

Comment:
the exact step that is displayed here depends on whether this is bare land or not.